### PR TITLE
EARTH-291: Simple block paragraph type.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- EARTH-291: Featured Blocks Paragraph Type

# Needed By (Date)
- July release

# Urgency
- Normal

# Steps to Test

1. Check out branch EARTH-291 in stanford_paragraph_types, stanford_components, and SE3_BLT
2. Run `drush cim -y`
3. Run `drush csim local -y`
4. Run `drush features-import-all -y`
5. Navigate to a complex page and create a new `Featured Blocks` paragraph type
6. Review display

# Associated Issues and/or People
- EARTH-291
- Contains the centered container logic that we might use for other sections going forward

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_paragraph_feat_blocks/stanford_paragraph_feat_blocks.info.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: 'Feature Blocks'
+description: 'Featured blocks with a section header.'
+type: module
+core: 8.x
+dependencies:
+  - ds
+  - entity_reference_revisions
+  - feature_blocks
```
Comment (pookmish) [inline modules/stanford_paragraph_feat_blocks/stanford_paragraph_feat_blocks.info.yml]: This prevented me from enabling.

**Review by pookmish — CHANGES_REQUESTED** [review]
I think the dependency is off. it calls for `feature_blocks` but we dont have that. If i remove that it enables and looks correct.

**Comment by sherakama** [comment]
Not sure where that dependency came from. Definitely not needed. Ok updated.

**Comment by sherakama** [comment]
@pookmish

**Review by pookmish — APPROVED** [review]

**Pull request merged**